### PR TITLE
GUVNOR-2559: Guided Decision Table Editor: Improve appearance (for Red Hat Summit) - Changed collapse/expand toggle; and fixed some Show Case issues

### PR DESCRIPTION
**Pull request by manstis**
- Changed collapse/expand toggle.
- Fixed some Wires Showcase rendering issues.
- Renamed a getter to be more consistent with what it returns.

### CODE REVIEW

**Comment by manstis** [comment]
**DO NOT MERGE - ISSUE FOUND. FIXING AS P/O THIS PR**

File: uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/demo/WiresGridsDemoPresenter.java [hunk 1]
```diff
@@ -126,9 +126,9 @@ public void setup() {
         this.gridWidget4 = makeGridWidget4();
 
         //Associations need to be set before the Grids are added to the View.
-        gridWidget1.getModel().getColumns().get( 9 ).setLink( gridWidget2.getModel().getColumns().get( 0 ) );
-        gridWidget2.getModel().getColumns().get( 3 ).setLink( gridWidget3.getModel().getColumns().get( 0 ) );
-        gridWidget3.getModel().getColumns().get( 1 ).setLink( gridWidget1.getModel().getColumns().get( 0 ) );
+        gridWidget1.getModel().getColumns().get( 9 ).setLink( gridWidget2.getModel().getColumns().get( 1 ) );
+        gridWidget2.getModel().getColumns().get( 3 ).setLink( gridWidget3.getModel().getColumns().get( 1 ) );
+        gridWidget3.getModel().getColumns().get( 1 ).setLink( gridWidget1.getModel().getColumns().get( 1 ) );
```
Comment (ederign) [inline uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/demo/WiresGridsDemoPresenter.java]: @manstis  I really don't link this long lines! could you please extract this to a method to make the meaning of this code clear for me?

**Comment by manstis** [comment]
@ederign Fixed..

**Comment by ederign** [comment]
+1 thanks @manstis

**Pull request merged**